scottdgaines/fitlit
# Feature/charts part 2

#### What's this PR do?
* Add function for weekly data chart (activity)
* Add helper function as part of weekly data display refactor
* re-download plus icon 

#### Where should the reviewer start?
* scripts.js

#### How should this be manually tested?
* open the browser
* check if image can be added to html

#### Any background context you want to provide?

#### Are there any issues?

#### Next steps:

#### Questions:
